text_adventure.py

```diff
@@ -0,0 +1,190 @@
+import random
+
+class GameObject:
+    verb_dict = {
+        "examine": "Examine a creature in the room.",
+        "hit": "Hit a creature in the room.",
+        "help": "Get help about the available commands.",
+        "heal": "Heal for an amount of damage.",
+        "kill": "Immediately kill the target.",
+        "status": "Get your own status."
+    }
+
+    @staticmethod
+    def get_input():
+        command = input(": ").split()
+        verb_word = command[0]
+        if verb_word in GameObject.verb_dict:
+            verb = getattr(GameObject, verb_word)
+        else:
+            print(f"Unknown verb {verb_word}")
+            print("For a list of commands type \"help\"")
+            return
+    
+        if len(command) >= 2:
+            noun_word = command[1]
+            print(verb(noun_word))
+        else:
+            print(verb("nothing"))
```

Comment:
you can use the https://docs.python.org/3/library/constants.html?highlight=none#None None object for this purpose